src/cf-buttons/usage.md

```diff
@@ -361,6 +362,26 @@ For accessibility reasons, use the semantic `<button>` instead of a link when po
 <input type="submit" value="Input Tag" class="a-btn a-btn__super active">
 ```
 
+### Full-width buttons on x-small screens
+
+_Reduce screen size to see these in action_
```

Comment:
Yeah, this is it's own issue I'd like to spend some time testing, but for the time being, I think our time is best spent getting the code organized/moved/trimmed and not on the documentation sites. We can re-vist those in a month or so.